vdocs/intro/index.md

```diff
@@ -265,7 +265,7 @@ To Deploy Mermaid:
 **To deploy mermaid without a bundler, one can insert a `script` tag with an absolute address and a `mermaidAPI` call into the HTML like so:**
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/mermaid@<MERMAID_VERSION>/dist/mermaid.min.js"></script>
 <script>
   mermaid.initialize({ startOnLoad: true });
 </script>
```

Comment:
Should we recommend using ESM instead? If we're planning on moving to ESM only at some point in the future, we might as well let people get used to using ESM.

```suggestion
<script type="module">
  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@9/dist/mermaid.esm.min.mjs';
  mermaid.initialize({ startOnLoad: true });
</script>
```

Reply:
I forgot the fact that we had an ESM build now. Was waiting for v10 release to update it. 🤣